src/physics.js

```diff
@@ -0,0 +1,58 @@
+import Vec2 from 'vec2';
+
+export default class Physics {
+    constructor(cameraBody, distanceFactor = 1.5, timeStep = 10, gravity = 9.82) {
+        this.cameraBody = cameraBody;
+        this.physicsBodyArr = new Array();
+        this.distanceFactor = distanceFactor;
+        this.timeStep = timeStep;
+        this.gravity = gravity; //units/(s^2)
+    }
+
+    addBody(physicsBody) {
+        this.physicsBodyArr.push(physicsBody);
+        this.cameraSort();
+    }
+
+    resolveView() {
+        this.cameraSort();
+
+        for (const physicsBody of this.physicsBodyArr) {
+            const physicsWindow = this.cameraBody.polygon.scale(this.distanceFactor, Vec2(this.cameraBody.position), true);
+            if (physicsWindow.containsPoint(Vec2(physicsBody.position))) {
+                this.resolveBody(physicsBody, physicsWindow);
+            }
+
+            physicsBody.inView = this.inView(physicsBody);
+        }
+
+        return this;
+    }
+
+    resolveBody(physicsBody) {
+        const tempBody = physicsBody.clone;
+
+        tempBody.velocity[1] -= this.gravity * this.timeStep / 1000;
```

Comment:
Instead of dividing by 1000 everywhere, suggest baking that into timeStep in the constructor OR dealing with units/ms instead.  Not sure which makes more sense.